lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -189,8 +195,10 @@ def configure(conf)
         # Check for specialized GCE environments (Managed VM or Dataflow).
         # TODO: Add config options for these to allow for running outside GCE?
         attributes = fetch_gce_metadata('instance/attributes/').split
-        if attributes.include?('gae_backend_name') &&
-           attributes.include?('gae_backend_version')
+        if @force_compute_service
```

Comment:
We don't need to fetch 'attributes' at all if this is true, so maybe omit the whole thing, i.e.

```
unless @force_compute_service
  attributes = ...
  if attributes ...
end
```

I'm not crazy about the name 'force_compute_service', although I can't come up with a better name.  It seems like we're really disregarding the more-specific service, but I'm not sure how to capture that succinctly.  I'm ok leaving it as is if we can't come up with a better name.